.github/workflows/update-copyright-notices.yml

```diff
@@ -0,0 +1,42 @@
+name: Update copyright notices
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Do we need to run this on PR close with the simplified logic?
I‘d prefer to run it once/twice a year. Preferably in January.

Reply:
Not sure, I'd still run it on every push/merge because each PR may introduce old headers again.

Indepdendent of the frequency we can drop the deletion logic if we are OK to permanently have an updateChangelogNotices branch in jamulussoftware/jamulus.

Reply:
Ok. Got your concern. I wouldn’t drop the deletion logic since it might clutter the repo. Maybe we‘d even need to ensure different names of the branch since conflicting history on a local branch and on GH might confuse people pulling the repo.

Reply:
Heh, that was the initial reason for the rather specific branch names. However, having those would clutter people's checkouts (at least when they check them out). I'd keep it like it is now (single branch name) as noone should usually check out those automatic repos. If they do, they'd hopefully know why. :)

Reply:
Yes. Great!